src/methods/generate/index.ts

```diff
@@ -24,7 +24,7 @@ export const generate = async (threshold = 3, keyCount = 5): Promise<GeneratedKe
   */
 
   // other members of the array ingredients used in the algorithm
-  for (let i = 0; i < threshold; i++) {
+  for (let i = 0; i < threshold - 1; i++) {
```

Comment:
why -1??

Reply:
Issue with 3/5 keys when a known key is provided